src/flows/components/header/Submit.tsx

```diff
@@ -91,6 +91,10 @@ export const Submit: FC = () => {
           } else if (pipe.type === 'queryBuilder') {
             const {aggregateFunction, bucket, field, measurement, tags} = pipe
 
+            if (!bucket) {
```

Comment:
shouldn't we be setting a default bucket? Also, what happens if a bucket's name is `''`

Reply:
the setting of the default bucket should be happening somewhere else (like in a dropdown or something). is it legal to have a bucket with no name? i feel like there would be more issues than this with that idea

Reply:
🤷🏽